### ISSUE TITLE
Replace Docs for Section 508 Compliant on ONRR.gov 

### ISSUE THREAD

**Issue by cmathers1** [ONRR/onrr.gov-site#1197]
Good morning, 

Please replace the following documents. Thank you!
[940706.pdf](https://github.com/ONRR/onrr.gov-site/files/7619683/940706.pdf)
[940103.pdf](https://github.com/ONRR/onrr.gov-site/files/7619685/940103.pdf)
[931208.pdf](https://github.com/ONRR/onrr.gov-site/files/7619687/931208.pdf)
[930917.pdf](https://github.com/ONRR/onrr.gov-site/files/7619688/930917.pdf)
[930628.pdf](https://github.com/ONRR/onrr.gov-site/files/7619689/930628.pdf)
[930503.pdf](https://github.com/ONRR/onrr.gov-site/files/7619690/930503.pdf)
[930202.pdf](https://github.com/ONRR/onrr.gov-site/files/7619691/930202.pdf)
[920727.pdf](https://github.com/ONRR/onrr.gov-site/files/7619692/920727.pdf)
[920416.pdf](https://github.com/ONRR/onrr.gov-site/files/7619693/920416.pdf)
[920122.pdf](https://github.com/ONRR/onrr.gov-site/files/7619694/920122.pdf)
[890421.pdf](https://github.com/ONRR/onrr.gov-site/files/7619695/890421.pdf)
[880930.pdf](https://github.com/ONRR/onrr.gov-site/files/7619696/880930.pdf)
[881003.pdf](https://github.com/ONRR/onrr.gov-site/files/7619697/881003.pdf)
[880204.pdf](https://github.com/ONRR/onrr.gov-site/files/7619698/880204.pdf)

**Comment by cthomasONRR** [comment]
@cmathers1 

The following documents failed the accessibility check. Could you please go back and fix these 2 documents?
- 881003 - Issues with page content, alt text, tables, lists, headings
- 880930 - Alt text needed

The other 12 documents are now uploaded to the ONRR site. Thanks so much for your help on this project!

**Comment by cmathers1** [comment]
Thank you. I am fixing these now and will resend.

Sincerely,

Christina

Christina Mathers
Staff Assistant
Director/Deputy Director's Office
Office of Natural Resources Revenue
6th Avenue and Kipling Street Bldg. 85
Denver, Colorado 80225
Phone:  303-231-3429
Cell: 303-503-1315
________________________________
From: Christine Thomas ***@***.***>
Sent: Tuesday, November 30, 2021 7:02 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Mathers, Christina ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Replace Docs for Section 508 Compliant on ONRR.gov (Issue #1197)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@cmathers1<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcmathers1&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C63b45648a9244ecf258808d9b40a21de%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637738777878943954%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=wQUIdBffZnlXro3uHLEINl%2BXBVu3y9aYf7%2FD%2BdNI1uA%3D&reserved=0>

The following documents failed the accessibility check. Could you please go back and fix these 2 documents?

  *   881003 - Issues with page content, alt text, tables, lists, headings
  *   880930 - Alt text needed

The other 12 documents are now uploaded to the ONRR site. Thanks so much for your help on this project!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F1197%23issuecomment-982664257&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C63b45648a9244ecf258808d9b40a21de%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637738777878953912%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=aMVr003y1%2BdOGF6P351WZqNE622t2oX6u70K5G7%2FcO0%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAU5GF7QRB37C4IVB25BADTDUOTKOVANCNFSM5I7S5CIQ&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C63b45648a9244ecf258808d9b40a21de%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637738777878953912%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=pP%2BFERkmhbzADUvrthDMQosChtejwgJjv36iPhUDwVY%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C63b45648a9244ecf258808d9b40a21de%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637738777878963866%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ymyRaVhJbkegqqGVWulYHjXvDyGZuFVkExHh0Bthn8s%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cchristina.mathers%40onrr.gov%7C63b45648a9244ecf258808d9b40a21de%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637738777878963866%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=GsAHvnt%2BMZbOCwj%2BEKI6OyPMf6XVT%2F6X94EJ3Vb%2Fs14%3D&reserved=0>.

**Comment by cthomasONRR** [comment]
@cmathers1 Thank you so much! I have the last 2 pdfs uploaded on the live site. Closing this issue.
